packages/tools/client-debugger/client-debugger-view/src/components/ContainerSummaryView.tsx

```diff
@@ -299,39 +304,39 @@ interface ActionsBarProps extends IContainerActions {
 function ActionsBar(props: ActionsBarProps): React.ReactElement {
 	const { isContainerConnected, tryConnect, forceDisconnect, closeContainer } = props;
 
-	const connectButtonTooltipId = useId("connect-button-tooltip");
-	const disconnectButtonTooltipId = useId("disconnect-button-tooltip");
-	const disposeContainerButtonTooltipId = useId("dispose-container-button-tooltip");
+	// const connectButtonTooltipId = useId("connect-button-tooltip");
+	// const disconnectButtonTooltipId = useId("disconnect-button-tooltip");
+	// const disposeContainerButtonTooltipId = useId("dispose-container-button-tooltip");
 
 	const changeConnectionStateButton = isContainerConnected ? (
-		<TooltipHost content="Disconnect Container" id={disconnectButtonTooltipId}>
-			<IconButton
-				onClick={forceDisconnect}
-				disabled={forceDisconnect === undefined}
-				menuIconProps={{ iconName: "PlugDisconnected" }}
-				aria-describedby={disconnectButtonTooltipId}
-			/>
-		</TooltipHost>
+		<Button
+			icon={<PlugDisconnected24Regular />}
+			onClick={forceDisconnect}
+			disabled={forceDisconnect === undefined}
+		>
+			{" "}
+			Disconnect Container{" "}
+		</Button>
 	) : (
-		<TooltipHost content="Connect Container" id={connectButtonTooltipId}>
-			<IconButton
-				onClick={tryConnect}
-				disabled={tryConnect === undefined}
-				menuIconProps={{ iconName: "PlugConnected" }}
-				aria-describedby={connectButtonTooltipId}
-			/>
-		</TooltipHost>
+		<Button
+			icon={<PlugConnected24Regular />}
+			onClick={tryConnect}
+			disabled={tryConnect === undefined}
+		>
+			{" "}
+			Connect Container{" "}
+		</Button>
 	);
 
 	const disposeContainerButton = (
-		<TooltipHost content="Close Container" id={disposeContainerButtonTooltipId}>
-			<IconButton
-				onClick={closeContainer}
-				disabled={closeContainer === undefined}
-				menuIconProps={{ iconName: "Delete" }}
-				aria-describedby={disposeContainerButtonTooltipId}
-			/>
-		</TooltipHost>
+		<Button
+			icon={<LockClosed24Filled />}
```

Comment:
Is there an icon for a trash can, like we had before? In this case "closing" means something like "throwing away" the container, more than "closing a lock".

Reply:
I was actually thinking the same thing for the "disposed" state. I wonder if the alert icon you're currently using for "disposed" would be better for "closed" and trashcan for "disposed"? I'd be interested in a designer's thoughts on this 🤣

Reply:
Changed to `Delete24Regular` icon.